### ISSUE TITLE
EC controlled entites are not leaving blocked state

### ISSUE THREAD

**Issue by mplogas** [danobot/entity-controller#195]
<<< Do not delete the following template headers. All information must be filled in because we need it to analyse the problem. Your issue will be auto closed without this information.
Before opening a new issue, please read the documentation and search for keywords relating to your problem. Since this is an open source project expect to be involved in the issue resolution because we depend on your support. >>>

## Description
EC controlled entites are not leaving blocked state. Weird thing is it worked for a while when i set it up initially. I updated HA and the plugin in between. Rolling back to 6.11 didn't change the behaviour, though.

## Configuration

arch | x86_64
-- | --
dev | false
docker | true
hassio | false
installation_type | Home Assistant Container
os_name | Linux
os_version | 4.15.18-24-pve
python_version | 3.8.5
timezone | Europe/Berlin
version | 0.115.1
virtualenv | false

EC: 6.11, 7.0, 8.0

```
bad_oben: 
  sensor: binary_sensor.bad_oben_sensor_presence
  sensor_resets_timer: true
  entity:  light.bad_oben_decke
  delay: 60
  start_time: sunset - 00:30:00
  end_time: sunrise + 00:30:00
  service_data:
    brightness_pct: 15
    kelvin: 2700
  night_mode:
    delay: 90
    service_data:
      brightness_pct: 5
      kelvin: 2000
    start_time: '01:00:00'
    end_time: '05:30:00'
```

## Steps to reproduce
1. Set up a light
2. Trigger the sensor
3. light turns on
4. ec stays blocked
5. turning the light off through lovelace/deconz puts its state back to idle

![grafik](https://user-images.githubusercontent.com/842121/94375437-e4d36e00-0113-11eb-8887-0970130453db.png)

### Expected behavior
EC unblocks after expiration timestamp when no movement is detected or the light state is overridden. 

### Actual Behaviour
EC remains blocked, thus not turning off the light. Setting ```grace_period``` to a higher value doesn't have any effect.

## Logs
```
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] sensor_state_change :: 'binary_sensor.bad_oben_sensor_presence' Sensor state change to: on
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] sensor_state_change :: state: 'idle'
2020-09-27 22:44:16 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=off; min_mireds=153, max_mireds=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:43:41.012480+02:00>
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are OFF.
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Exiting idle
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_exit_idle | Performing Transition Behaviour
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_exit_idle | Action - ignore
2020-09-27 22:44:16 INFO (MainThread) [transitions.core] Exited state idle
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Entering active
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] NIGHT MODE ENABLED: OrderedDict([('delay', 90), ('service_data', OrderedDict([('brightness_pct', 5), ('kelvin', 2000)])), ('start_time', '01:00:00'), ('end_time', '05:30:00')])
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] now_is_between start time 2020-09-27 01:00:00.905362+02:00
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] now_is_between end time 2020-09-27 05:30:00.905362+02:00
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Using DAY MODE parameters: {'delay': 60, 'service_data': OrderedDict([('brightness_pct', 15), ('kelvin', 2700)]), 'service_data_off': None}
2020-09-27 22:44:16 INFO (MainThread) [custom_components.entity_controller.bad_oben] _start_timer :: Light params: {'delay': 60, 'service_data': OrderedDict([('brightness_pct', 15), ('kelvin', 2700)]), 'service_data_off': None}
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_enter_active :: light params before turning on: {'delay': 60, 'service_data': OrderedDict([('brightness_pct', 15), ('kelvin', 2700)]), 'service_data_off': None}
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_enter_active | Performing Transition Behaviour
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_enter_active | Action - on
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_enter_active | Performing Action - Turning on
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] turn_on_control_entities :: Turning on light.bad_oben_decke with service parameters OrderedDict([('brightness_pct', 15), ('kelvin', 2700)])
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] call_service :: Calling service turn_on on light.bad_oben_decke
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] call_service :: Setting ignore_state_changes_until to 2020-09-27 22:44:18.913645
2020-09-27 22:44:16 INFO (MainThread) [transitions.core] Entered state active_timer
2020-09-27 22:44:16 INFO (MainThread) [transitions.core] Entered state active
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: [light.bad_oben_decke] - old: <state light.bad_oben_decke=off; min_mireds=153, max_mireds=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:43:41.012480+02:00>, new: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>
2020-09-27 22:44:16 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: This state change is within 2 seconds of calling a service. Ignoring this state change because its probably caused by EC itself.
2020-09-27 22:44:17 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: [light.bad_oben_decke] - old: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>, new: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=10, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>
2020-09-27 22:44:17 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: A significant attribute changed and will be handled
2020-09-27 22:44:17 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: This state change is within 2 seconds of calling a service. Ignoring this state change because its probably caused by EC itself.
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: [light.bad_oben_decke] - old: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=10, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>, new: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: A significant attribute changed and will be handled
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: We are in active timer and the state of observed state entities changed.
2020-09-27 22:44:23 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are ON. [light.bad_oben_decke]
2020-09-27 22:44:23 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are ON. [light.bad_oben_decke]
2020-09-27 22:44:23 INFO (MainThread) [transitions.core] Exited state active_timer
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Exiting active
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_exit_active :: Turning off entities, cancelling timer
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_exit_active | Performing Transition Behaviour
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_exit_active | Action - ignore
2020-09-27 22:44:23 INFO (MainThread) [transitions.core] Exited state active
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Entering blocked
2020-09-27 22:44:23 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are ON. [light.bad_oben_decke]
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_enter_blocked | Performing Transition Behaviour
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] on_enter_blocked | Action - ignore
2020-09-27 22:44:23 INFO (MainThread) [transitions.core] Entered state blocked
2020-09-27 22:44:23 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are ON. [light.bad_oben_decke]
2020-09-27 22:44:30 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] sensor_state_change :: 'binary_sensor.bad_oben_sensor_presence' Sensor state change to: off
2020-09-27 22:44:30 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] sensor_state_change :: state: 'blocked'
```

## Version
v6.11, 7.0 and 8.0 is affected.

**Comment by mplogas** [comment]
Update: moving back to 6.0 and everything works as expected.

6.0 log, same setup as above:

```
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] binary_sensor.bad_oben_sensor_presence Sensor state change to: on
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state: idle
2020-09-27 23:23:38 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=off; min_mireds=153, max_mireds=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:22:53.019210+02:00>
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are OFF.
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Exiting idle
2020-09-27 23:23:38 INFO (MainThread) [transitions.core] Exited state idle
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] NIGHT MODE ENABLED: OrderedDict([('delay', 90), ('service_data', OrderedDict([('brightness_pct', 5), ('kelvin', 2000)])), ('start_time', '01:00:00'), ('end_time', '05:30:00')])
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] now_is_between start time 2020-09-27 01:00:00.323669+02:00
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] now_is_between end time 2020-09-27 05:30:00.323669+02:00
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Using DAY MODE parameters: {'delay': 60, 'service_data': OrderedDict([('brightness_pct', 15), ('kelvin', 2700)]), 'service_data_off': None}
2020-09-27 23:23:38 INFO (MainThread) [custom_components.entity_controller.bad_oben] {'delay': 60, 'service_data': OrderedDict([('brightness_pct', 15), ('kelvin', 2700)]), 'service_data_off': None}
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] light params before turning on: {'delay': 60, 'service_data': OrderedDict([('brightness_pct', 15), ('kelvin', 2700)]), 'service_data_off': None}
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Turning on light.bad_oben_decke with service parameters OrderedDict([('brightness_pct', 15), ('kelvin', 2700)])
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] Calling service light.bad_oben_decke turn_on
2020-09-27 23:23:38 INFO (MainThread) [transitions.core] Entered state active_timer
2020-09-27 23:23:38 INFO (MainThread) [transitions.core] Entered state active
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: [light.bad_oben_decke] - old: <state light.bad_oben_decke=off; min_mireds=153, max_mireds=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:22:53.019210+02:00>, new: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>
2020-09-27 23:23:38 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are ON. [light.bad_oben_decke]
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: [light.bad_oben_decke] - old: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>, new: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=10, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] significant attribute only change
2020-09-27 23:23:38 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=10, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>
2020-09-27 23:23:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are ON. [light.bad_oben_decke]
2020-09-27 23:23:44 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: [light.bad_oben_decke] - old: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=10, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>, new: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>
2020-09-27 23:23:44 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] significant attribute only change
2020-09-27 23:23:44 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>
2020-09-27 23:23:44 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] State entities are ON. [light.bad_oben_decke]
2020-09-27 23:23:52 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] binary_sensor.bad_oben_sensor_presence Sensor state change to: off
2020-09-27 23:23:52 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state: active_timer
2020-09-27 23:24:38 DEBUG (Thread-35) [custom_components.entity_controller.bad_oben] Timer expired
2020-09-27 23:24:38 INFO (Thread-35) [transitions.core] Exited state active_timer
2020-09-27 23:24:38 DEBUG (Thread-35) [custom_components.entity_controller.bad_oben] Turning off entities, cancelling timer
2020-09-27 23:24:38 INFO (Thread-35) [transitions.core] Exited state active
2020-09-27 23:24:38 DEBUG (Thread-35) [custom_components.entity_controller.bad_oben] Entering idle
2020-09-27 23:24:38 DEBUG (Thread-35) [custom_components.entity_controller.bad_oben] Turning off light.bad_oben_decke
2020-09-27 23:24:38 DEBUG (Thread-35) [custom_components.entity_controller.bad_oben] Calling service light.bad_oben_decke turn_off
2020-09-27 23:24:38 DEBUG (Thread-35) [custom_components.entity_controller] Resetting state
2020-09-27 23:24:38 INFO (Thread-35) [transitions.core] Entered state idle
2020-09-27 23:24:38 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: [light.bad_oben_decke] - old: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:23:38.360983+02:00>, new: <state light.bad_oben_decke=off; min_mireds=153, max_mireds=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T23:24:38.757655+02:00>
```

**Comment by danobot** [comment]
Hi @mplogas ,
thanks for the excellent issue description! Have never seen a better example!
Your problem is likely caused by delays between `turn_on` service call and when the light is actually turned on. (latency) 

Currently there exists a 2 second `grace_period` (configurable) to ignore state updates during this time period between service call and light actually changing state. Grace period is the old way of achieving this (its very janky). The new method is using HA Context API, which I implemented for issue #192 and pending to be released.

Please test using the EC version on [branch issue/192](https://github.com/danobot/entity-controller/tree/issue/192) and let me know :)

**Comment by mplogas** [comment]
Hi @danobot ,
thanks! I'm a developer myself, I just write my issues the way I would like to receive them :)

Anyhow, I installed the component from issue/192. (I delete the component in HACS and moved the content of the custom_component into the corresponding folder and restarted. Is that the right procedure?) However it seems to go into blocked again

```
2020-09-28 19:54:00 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=off; min_mireds=153, max_mireds=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-28T19:53:39.202484+02:00>
2020-09-28 19:54:00 INFO (MainThread) [transitions.core] Exited state idle
2020-09-28 19:54:00 INFO (MainThread) [custom_components.entity_controller.bad_oben] _start_timer :: Light params: {'delay': 60, 'service_data': OrderedDict([('brightness_pct', 15), ('kelvin', 2700)]), 'service_data_off': None}
2020-09-28 19:54:00 INFO (MainThread) [transitions.core] Entered state active_timer
2020-09-28 19:54:00 INFO (MainThread) [transitions.core] Entered state active
2020-09-28 19:54:07 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-28T19:54:00.059841+02:00>
2020-09-28 19:54:07 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-28T19:54:00.059841+02:00>
2020-09-28 19:54:07 INFO (MainThread) [transitions.core] Exited state active_timer
2020-09-28 19:54:07 INFO (MainThread) [transitions.core] Exited state active
2020-09-28 19:54:07 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-28T19:54:00.059841+02:00>
2020-09-28 19:54:07 INFO (MainThread) [transitions.core] Entered state blocked
2020-09-28 19:54:07 INFO (MainThread) [custom_components.entity_controller.bad_oben] <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-28T19:54:00.059841+02:00>
```

I'm not a Python dev myself, so I'm afraid my help is limited to reporting. I'll switch back to 6.0 for now. Let me know when there's anything to test, it'S the least I can do.

Thanks again.

**Comment by danobot** [comment]
Is this behaviour erratic or consistent? Can you reproduce it using a basic EC configuration? (What is the minimal configuration that reproduces the issue)?

```
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: [light.bad_oben_decke] - old: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=10, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>, new: <state light.bad_oben_decke=on; min_mireds=153, max_mireds=370, brightness=38, color_temp=370, is_deconz_group=False, friendly_name=Bad Oben Decke, supported_features=43 @ 2020-09-27T22:44:16.947075+02:00>
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: A significant attribute changed and will be handled
2020-09-27 22:44:23 DEBUG (MainThread) [custom_components.entity_controller.bad_oben] state_entity_state_change :: We are in active timer and the state of observed state entities changed.
```

These logs tell me that something controlled the entity, causing the state change to blocked. Do you use circadian lighting? Any other automation interacting with your entity?

**Comment by mplogas** [comment]
Hi @danobot,
It seems the issue/192 resolved the blocked state eventually. I cannot reproduce the issue with this branch anymore. Thanks!

**Comment by danobot** [comment]
ok cool, I'll release that now. Please consider donating or supporting the project in the future. Thanks :)
